docs/HTTP-batchsink.md

```diff
@@ -0,0 +1,61 @@
+# HTTP Sink
+
+
+Description
+-----------
+Sink plugin to send the messages from the pipeline to an external http endpoint.
+
+Properties
+----------
+
+**url:** The URL to post data to.
+
+**method:** The HTTP request method. Defaults to POST.
+
+**bathSize:** Batch size. Defaults to 1.
+
+**messageFormat:** Format to send messsage in. Options are JSON,Form,Custom. Defaults to JSON.
+
+**body:** Custom message. This would be mandatory if message format is selected as Custom.
+          In case user wants to have Custom message and want to use some of the input record fields as variables to build the message,
```

Comment:
this is worded very poorly. How does a user add fields from the message to the custom post payload? can you also add an example?